### PR TITLE
Right-click to remove carbon-copies

### PR DESCRIPTION
**Pull request by mozi-h**
## About The Pull Request
- Enforce Right-click to tear off a carbon copy instead of any click.
- Add examine text to explain this

## Why It's Good For The Game
Prevents accidental premature ripping off of carbon copies. HoPs hate(d) this!
Also [improves](https://github.com/tgstation/tgstation/pull/57289#discussion_r585076647) [consistency](https://github.com/tgstation/tgstation/pull/57400#issue-584302253).

## Changelog
:cl: mozi_h
qol: Carbon-copies can now only be removed with a right-click. HoPs, you are welcome (yet again)!
/:cl:

### CODE REVIEW

**Comment by RaveRadbury** [comment]
Paperwork hero

**Comment by mozi-h** [comment]
> Paperwork hero

My work is not done yet!
grafik

**Comment by maxymax13** [comment]
do "much needed paperwork improvement" prs count as hop buffs

**Comment by itseasytosee** [comment]
Buffing paperwork is not a quality of life for bureaucrats.

File: code/modules/paperwork/carbonpaper.dm [hunk 1]
```diff
@@ -39,7 +45,7 @@
 		user.put_in_hands(Copy)
 
 /obj/item/paper/carbon/attack_hand(mob/living/user, list/modifiers)
```
Comment (Mothblocks) [inline code/modules/paperwork/carbonpaper.dm:47]: Change this to `/attack_hand_secondary`, perform your behavior, then return CANCEL_ATTACK_CHAIN.

**Review by Mothblocks — CHANGES_REQUESTED** [review]

File: code/modules/paperwork/carbonpaper.dm (hunk 1 above)
Comment (Mothblocks) [reply code/modules/paperwork/carbonpaper.dm:47]: Or rather, add `/attack_hand_secondary` instead of using modifiers.

**Comment by Mothblocks** [comment]
Undraft if ready.

**Pull request merged**